processing/data_collection/gazette/pipelines.py

```diff
@@ -126,6 +119,52 @@ def _is_file_type(self, filepath, file_types):
         return self.get_file_type(filepath) in file_types
 
 
+class PostgreSQLPipeline:
```

Comment:
As the default DB in the repo is SQLite, why not `SQLDatabasePipeline` or something similar?

Reply:
It makes sense. I renamed it!